modules/displayUtils.js

```diff
@@ -0,0 +1,46 @@
+export const displayList = () => {
+  const list = document.getElementById('list');
+  const addNew = document.getElementById('addnew');
+  const contact = document.getElementById('contact');
+  const bookSection = document.getElementById('books');
+  const createBook = document.getElementById('createBook');
+  const contactInfo = document.getElementById('contactinfo');
+  createBook.style.display = 'none';
+  bookSection.style.display = 'flex';
+  contact.style.color = 'black';
+  addNew.style.color = 'black';
+  list.style.color = 'blue';
+  contactInfo.style.display = 'none';
+};
+
+export const displayForm = () => {
+  const list = document.getElementById('list');
+  const addNew = document.getElementById('addnew');
+  const contact = document.getElementById('contact');
+  const bookSection = document.getElementById('books');
+  const createBook = document.getElementById('createBook');
+  const contactInfo = document.getElementById('contactinfo');
+  const hr = document.getElementById('hr');
+  createBook.style.display = 'flex';
+  contact.style.color = 'black';
+  addNew.style.color = 'blue';
+  list.style.color = 'black';
+  bookSection.style.display = 'none';
+  contactInfo.style.display = 'none';
+  hr.style.display = 'none';
+};
+
+export const displayContact = () => {
+  const list = document.getElementById('list');
+  const addNew = document.getElementById('addnew');
+  const contact = document.getElementById('contact');
+  const bookSection = document.getElementById('books');
+  const createBook = document.getElementById('createBook');
+  const contactInfo = document.getElementById('contactinfo');
+  contact.style.color = 'blue';
+  addNew.style.color = 'black';
```

Comment:
- [ ] Great job so far. Please kindly create a new module and ensure the current date is displayed below the navigation bar, for this requirement, as stated you should use the [Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) library. See [Get the current date and time](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time)

- kindly ensure Package is added as a .js file to the project (in full or minified version)
- Module is imported `import { DateTime } from "luxon";`